capellambse/loader/core.py

```diff
@@ -260,8 +260,8 @@ def add_namespace(self, name: str, uri: str) -> None:
         fragroot = self.root
         olduri = fragroot.nsmap.get(name)
         if olduri is not None and olduri != uri:
-            raise ValueError(
-                f"Namespace {name!r} already registered with URI {olduri!r}"
+            LOGGER.warning(
+                "Namespace %r already registered with URI %r", name, olduri
```

Comment:
> Specifically, this change allows the URI for a given namespace to be updated if necessary, without causing an error to be raised.

Wouldn't this actually cause even more trouble, by putting e.g. a 5.0 namespace into a 6.0 model? Did you try loading such a model in Capella after this change?

Reply:
So, I just tried it, and yes: If you inject a 5.0 namespace into a 6.0 model, the entire model breaks. So we at least need to `return` early here to prevent overwriting a good namespace with a bad one.

Reply:
Yes, I agree that overwriting the existing namespace is not the correct way here. I added a return, so in the case that an namespace exists, it is no longer overwritten.